torch/csrc/jit/python/pybind_utils.h

```diff
@@ -696,13 +697,7 @@ inline py::object toPyObject(IValue ivalue) {
   } else if (ivalue.isDevice()) {
     return py::cast<py::object>(THPDevice_New(std::move(ivalue).toDevice()));
   } else if (ivalue.isGenericDict()) {
-    auto dict = std::move(ivalue).toGenericDict();
-    py::dict py_dict;
-    for (auto& pair : dict) {
-      py_dict[toPyObject(IValue{pair.key()})] =
-          toPyObject(IValue{pair.value()});
-    }
-    return std::move(py_dict);
+    return py::cast(ScriptDict(ivalue));
```

Comment:
Curious if this has caused any issues in internal CI.

Reply:
It has. I didn't check them all, but I think they should be addressable.